conf/deployment/fusion_hci_pc/hypershift_client_bm_2w.yaml

```diff
@@ -0,0 +1,10 @@
+ENV_DATA:
+  platform: 'hci_baremetal'
```

Comment:
we should refrain from using the 'HCI' term, since we are not actually installing on Fusion HCI in particular, and once we do, this might cause confusion.
I understand this already exists in our repo so we can address this in a separate PR

Reply:
We have a separate ticket for this where we introduce another platform type. @ebondare assigned to it. We need to change it with one PR across all the modules. If it is done only here it would not be in use.